hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/UpgradeDowngrade.java

```diff
@@ -205,4 +205,9 @@ protected Map<ConfigProperty, String> downgrade(HoodieTableVersion fromVersion,
       throw new HoodieUpgradeDowngradeException(fromVersion.versionCode(), toVersion.versionCode(), false);
     }
   }
+
+  public HoodieTableMetaClient getMetaClient() {
+    metaClient.reloadActiveTimeline();
```

Comment:
Should we move `reloadActiveTimeline` inside the run method?

Reply:
Done.